js/script-api-changes.js

```diff
@@ -172,15 +202,21 @@ async function loadApiChanges(language) {
     initSupportedTable("RAIL", translations[language].rail.toUpperCase(), language);
     initSupportedTable("ROAD", translations[language].road.toUpperCase(), language);
 
-    const marineApi = await loadApiDescription("MARINE");
-    const railApi = await loadApiDescription("RAIL");
-    const roadApi = await loadApiDescription("ROAD");
-
-    populateDeprecations(JSON.parse(marineApi), "MARINE");
-    populateDeprecations(JSON.parse(railApi), "RAIL");
-    populateDeprecations(JSON.parse(roadApi), "ROAD");
-
-    populateSupported(JSON.parse(marineApi), "MARINE");
-    populateSupported(JSON.parse(railApi), "RAIL");
-    populateSupported(JSON.parse(roadApi), "ROAD");
+    const marineApi = await loadApiDescription("MARINE").then(function (response) {
```

Comment:
Promise all hakee kaikista samanaikaisesti
```suggestion
const [marineApi, railApi, roadApi] = await Promise.all([loadApiDescription("MARINE"), loadApiDescription("RAIL"), loadApiDescription("ROAD")])
    .then(apis => apis.map(a => JSON.parse(a)))
```